code/modules/food_and_drinks/drinks/drinks.dm

```diff
@@ -91,6 +91,8 @@
 		to_chat(user, "<span class='notice'>You transfer [trans] units of the solution to [target].</span>")
 
 		if(iscyborg(user)) //Cyborg modules that include drinks automatically refill themselves, but drain the borg's cell
+			if (type == /obj/item/reagent_containers/food/drinks/drinkingglass) // Not all containers are refillable
```

Comment:
This is a bad fix.

If I understand the bug correctly, drinking glasses are only special because the borg creates them, yes? If so, have borg-created drinking glasses have a trait to say they are borg-created, and check that instead.